config/routes.rb

```diff
@@ -775,10 +775,6 @@
       to: 'users#fill_user_select',
       as: 'fill_user_select'
 
-  get 'users/list',
-      to: 'users#list',
-      as: 'list_users'
-
```

Comment:
What does this have to do with this PR?

Reply:
This route is out of use so I deleted it. We can put that in another pR if you want.

Reply:
Yes, I've seen you already did this [here](https://github.com/MaMpf-HD/mampf/commit/de9e1b102d7c4a53b92644ae778cbbb301b13cbd#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5L778-L781), so no need to have it here.